hamp_radar/flight_dataset.py

```diff
@@ -0,0 +1,125 @@
+from collections.abc import Iterable
+from pathlib import Path
+from typing import Optional
+import xarray as xr
+import numpy as np
+
+from iquick import extract_raw_arrays
+from decoders import decode_time, pds_decode
+from geometries import FlightGeometry, DatasetGeometry
+
+from serde_flight import get_flight_geometry
+
+
+def postprocess_iq(ds):
+    # TODO(ALL): move to new file
+    return ds.pipe(decode_time)
+
+
+def read_datasetblock(
+    datablock: DatasetGeometry,
+    radar_tag: Optional[str] = None,
```

Comment:
The `radar_tag` should already be part of the `mainblock` so we should not pass it in again a second time.

Reply:
yes that's true but I thought it was clearer like this, although yes it's more error prone, will change...

Reply:
better now?